### DISCUSSION TITLE
On hover of the q-table q-td element, displaying a q-card

### DISCUSSION THREAD

**Discussion by ayazici-vekend** [quasarframework/quasar#9257] [General - Components / Directives / etc]
I have a q-table and on one of the columns i display an image. What i want is that when i hover on that image, i would like to show the details of that product in a q-card.
I implemented the hover q-card as the child of the q-img, and q-img's is child of q-td. When i hover q-card is displayed as bound to the q-img, meaning it only shows the part as much as it fits inside the block, so only like quarter of the card is being shown, and i want to display it fully visible.
I tried overflow: visible on the q-img and q-td and z-index: 1000 on q-card, but both did not work.
P.S: When i make the position of the q-card fixed, it shows the full card, but then i lose dynamicism on scroll which is not something i want.

**Comment by ontwikkelfabriek** [comment]
If you can create a codepen.io that demo's what you have now we can provide better support. 
You can easily create a codepen by using one of quasar's example codepens.

**Reply by ayazici-vekend** [reply]
https://codepen.io/yaziciahmet/pen/yLMOyrp
This is what i have at the moment, and i dont know why but background property is not working in codepen.
I want my detailed card to be fully shown when i hover over the image i have. And also i added a logic to make sure only the hovered image's card is shown by adding a data variable hoveredItem so only the hoveredItem's detailed card will be prompted. It was working on my personal computer, but not working here and i am not sure why? It would be awesome if you could also check that out.
